### PR TITLE
docs: add documentation for cli commands

### LINKED ISSUE #126: docs: document instance entrypoints/migrate-secret-key cli commands
The two CLI commands below are not documented:

```
inveniomanage instance entrypoints
inveniomanage instance migrate-secret-key
```

Especially secret key migration would need docs - i.e. how should other modules hook into the secret key migration.

Check also API documentation for missing modules.

### PR DESCRIPTION
**Pull request by zzacharo**
closes #126

### CODE REVIEW

File: invenio_base/__init__.py [hunk 1]
```diff
@@ -174,6 +174,80 @@ def index():
...
+Migrating the application's old secret key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``instance migrate_secret_key`` subcommand helps you migrate your
+application's old secret key:
+
+.. code-block:: console
+
+   $ inveniomanage instance migrate_secret_key --old-key <old_key>
+
+The purpose of this command is to provide the user the capability to change the
```
Comment (dinosk) [inline invenio_base/__init__.py]: I would write the administrator instead of user.

File: invenio_base/__init__.py [hunk 2]
```diff
@@ -174,6 +174,80 @@ def index():
...
+Invenio application's secret_key and migrate that change in all database's
+EncryptedType properties through an entrypoint group called
+`invenio_base.secret_key'`. There you can specify your migration function that
+will receive the old secret_key that can be used to decrypt the old properties
+and encrypt them again with the application's new secret_key.
+
+You can register your migration function as below in your package's entrypoints
```
Comment (dinosk) [inline invenio_base/__init__.py]: Maybe `as shown below`?

File: invenio_base/__init__.py [hunk 3]
```diff
@@ -174,6 +174,80 @@ def index():
...
+           '<entrypoint_name> = <entrypoint_function>'
+       ]
+   }
+
+Also you can see an example of use in `invenio_oauthclient
+<https://github.com/inveniosoftware/invenio-oauthclient>`_
+package's setup.py.
+
+.. note::
+ You should change your application's `secret_key` before calling the migration
```
Comment (dinosk) [inline invenio_base/__init__.py]: change your application's `secret_key` (in the config) before calling ...?

**Comment by zzacharo** [comment]
@dinosk I made the changes.

**Pull request merged**